client/src/utils/constants.js

```diff
@@ -1,5 +1 @@
 // Store constants here for reusability across application
-import dotenv from "dotenv";
-dotenv.config();
```

Comment:
this should be fine if we have it in the top level of the app, if we don't already, we should also add to the sample env file so other devs will know what to fill in locally

Reply:
@jwu910 I'm not sure if my message was never sent last time but I didn't get a response to what the top level of the app is, would we just add it to index.js?

Reply:
https://github.com/BobaTalks/internship-tracker/blob/main/client/src/index.jsx

I would say index.jsx is the `entry point` where we can import this